_assets/javascripts/components/bitmovin.js

```diff
@@ -204,6 +204,9 @@ class BitmovinManager {
   }
 
   onPlayerStart() {
+    var defJSLoaded = new Event('mediaMetricInteraction');
+    document.dispatchEvent(defJSLoaded);
```

Comment:
you migh want to rename this var. I know this is from when I sent you an example. defJSLoaded is referencing "deferred javascript file loaded".